en/services/ftp.md

```diff
@@ -209,47 +209,50 @@ On completion of the burst (or the file), if there are any missing parts of the
 -->
 
 The sequence of operations is:
 1. GCS (client) sends [OpenFileRO](#OpenFileRO) command specifying the file path to open.
    - The payload must specify: `data[0]`= file path string, `size`=length of file path string.
-1. Drone (server) responds with either 
+1. Drone (server) responds with either
    - ACK on success. The [payload](#payload) must specify fields: `session` = file session id, `size` = 4, `data` = length of file that has been opened.
-   - NAK with [error information](#error_codes), e.g. `NoSessionsAvailable`, `FileExists`. 
+   - NAK with [error information](#error_codes), e.g. `NoSessionsAvailable`, `FileExists`.
      The GCS may cancel the operation, depending on the error.
-1. Client (i.e. GCS) sends [BurstReadFile](#BurstReadFile) command specifying the part of the file that it wants to get.
-   - The payload must specify: `session`: the current session id, `offset` = offset in file of start of burst, `data`= length of burst, `size`=4.
-1. Server (drone) responds with either 
-   - ACK on success. The [payload](#payload) must specify fields: `session` = file session id, `size` = 4, `data` = lenght of file in burst. 
+1. Client (i.e. GCS) sends [BurstReadFile](#BurstReadFile) command specifying the part of the file that it wants to get from an offset to the end, along with the default size of each burst payload.
+   - The payload must specify: `session`: the current session id, `offset` = offset in file of start of burst, `size`= default length of payload in burst responses, `data` is ignored.
+1. Server (drone) responds with either
+   - ACK on success. The [payload](#payload) must specify fields: `session` = file session id, `size` = 4, `data` = length of file in burst.
    - NAK with [error information](#error_codes). The client may cancel the operation, depending on the error.
 1. Server sends stream of [BurstReadFile](#BurstReadFile) data to client (without ACK) until the whole burst is sent, or a server-defined burst size limit is reached.
-   - The payload must specify: `session`=current session, `size`=size of data to read, `offset`= position in original data of current chunk.
+   - The payload must specify: `session`=current session, `size`=size of data to read per burst message (max equal to payload size = 239), `offset`= position in original data of current chunk.
    - The payload must specify `burst_complete=0` for all chunks, except the last one, which must set `burst_complete=1`.
 1. Client repeats the `BurstReadFile` cycle at different offsets until the whole file is retrieved.
 1. The client must maintain its own record of the received (and missing) chunks.
-   It may request any missing chunks at either the end of a  burst or at the end of the file. 
-   Missing chunks can be requested using either `BurstReadFile` or `ReadFile`.
+   It may request any missing chunks at either the end of a burst or at the end of the file.
+   Missing chunks can be requested using either `ReadFile`.
```

Comment:
```suggestion
   It may request any missing chunks at either the end of a burst or at the end of the file.
   Missing chunks can be requested using `ReadFile`.
```